pr_insight/algo/utils.py

```
@@ -968,45 +972,28 @@
-        'skip_keys', 
-        'app_id', 
-        'redirect',
-        'trial_prefix_message', 
-        'no_eligible_message', 
-        'identity_provider', 
-        'ALLOWED_REPOS',
-        'APP_NAME'
-    }
-
-    # Get extra skip keys from settings
+    skip_keys = ['ai_disclaimer', 'ai_disclaimer_title', 'ANALYTICS_FOLDER', 'secret_provider', "skip_keys", "app_id", "redirect",
```

Ruff

Line too long (130 > 120)

(E501)